includes/header.php

```diff
@@ -4,6 +4,40 @@
     <meta charset="utf-8">
     <title>Treat Street</title>
     <link href="/styleguide/styles.css" rel="stylesheet">
```

Comment:
One last thing. Could you guys add this code above the styles.css <link> please? :)

`<link href="https://fonts.googleapis.com/css?family=Josefin+Sans:300,700i&display=swap" rel="stylesheet">`

Reply:
Okay. Sorry for the delay. The link has been added.

Reply:
Thanks!